packages/react-scripts/scripts/build.js

```diff
@@ -103,8 +103,8 @@ checkBrowsers(paths.appPath, isInteractive)
       // The SSR config still omits a css file - it's not yet possible to omit
       // file output in ExtractTextPlugin. This is not needed so lets clean
       // it up to avoid confusion.
-      const ssrCssPath = path.join(paths.appBuild, 'ssr.css');
-      const ssrCssMapPath = path.join(paths.appBuild, 'ssr.css.map');
+      const ssrCssPath = path.join(paths.appBuildSsr, 'ssr.css');
+      const ssrCssMapPath = path.join(paths.appBuildSsr, 'ssr.css.map');
```

Comment:
Same for these

Reply:
This part comes from #95 , and it won't bring any breaking even for build path(s).